.gitmodules

```diff
@@ -13,3 +13,7 @@
 [submodule "third_party/googletest"]
 	path = third_party/googletest
 	url = https://github.com/google/googletest.git
+[submodule "RIOT"]
+	path = partial/extern/RIOT
+	url = git@github.com:OYTIS/RIOT.git
```

Comment:
Should be an https url, so that others can freely clone it.

Reply:
Oops. Thank you!